### DISCUSSION TITLE
Duda parte 2

### DISCUSSION THREAD

**Discussion by Agustinuc** [IIC2133-PUC/2024-1#133] [Tarea 2] [answered]
holaa, tengo una duda sobre que tipo de algoritmo se debe ocupar para realizar esta parte de la tarea. En el enunciado dice que se debe Diseñar una estrategia de backtracking para solucionar un problema de restricciones. Esto seria para la parte 2? Y sino que tipo de algortimo se pide para realizar esta parte. Gracias

**Comment by sebacarrascop** [comment, marked as answer]
hola, si, para esta parte se espera uso de backtracking
